planner/core/indexmerge_path.go

```diff
@@ -560,10 +563,16 @@ func (ds *DataSource) generateIndexMergeJSONMVIndexPath(normalPathCnt int, filte
 					continue
 				}
 				vals = append(vals, v)
-			case ast.JSONOverlaps: // (json_overlaps(a->'$.zip', '[1, 2, 3]')
-				continue // TODO: support json_overlaps
 			case ast.JSONContains: // (json_contains(a->'$.zip', '[1, 2, 3]')
-				continue // TODO: support json_contains
+				indexMergeIsIntersection = true
+				fallthrough
+			case ast.JSONOverlaps: // (json_overlaps(a->'$.zip', '[1, 2, 3]')
```

Comment:
`json_overlaps(a, b)` is equal to `json_overlaps(b, a)`, that should be considered

Reply:
Updated. PTAL @xiongjiwei